core/src/queue.rs

```diff
@@ -6,7 +6,7 @@ use crossbeam_queue::ArrayQueue;
 use dashmap::{mapref::entry::Entry, DashMap};
 use eyre::{Report, Result};
 use iroha_crypto::HashOf;
-use iroha_data_model::transaction::VersionedTransaction;
+use iroha_data_model::prelude::*;
```

Comment:
Not sure if this really is clearer. Do we only use `VersionedTransaction` from prelude?

Reply:
ok